doc/util/chpl2rst.py

```diff
@@ -71,11 +71,16 @@ def gen_link(link, chapelfile):
     abspath = os.path.realpath(chapelfile)
     filename = os.path.split(chapelfile)[1]
 
-    chpl_home = os.path.realpath(os.getenv('CHPL_HOME'))
+    chpl_home = os.getenv('CHPL_MAKE_HOME')
+    if not chpl_home:
+        chpl_home = os.getenv('CHPL_HOME')
+
     if not chpl_home:
         print('Error: --link flag only works when $CHPL_HOME is defined')
         sys.exit(1)
-    elif not chpl_home in abspath:
+
+    chpl_home = os.path.realpath(chpl_home)
```

Comment:
Another optional recommendation: 

It could be clearer if we used a different variable before this line, like `chpl_home_env`, such that this is:

```python
chpl_home = os.path.realpath(chpl_home_env)
```